kalite/distributed/hbtemplates/exercise/software-keyboard.handlebars

```diff
@@ -0,0 +1,13 @@
+<button class='simple-button orange' id='show-keyboard'> {{_ "Hide Keypad" }}</button>
+
+<div class='container-fluid' id='software-keyboard'>
+    {{#each keys }}
+        <div class="row">
+            <div class="col-xs-12">
```

Comment:
Tried it. Got this:
![image](https://cloud.githubusercontent.com/assets/1680573/6448045/1407f63e-c0cc-11e4-9a44-637a99aca471.png)